### PR TITLE
Create component task, adding instructions to CONTRIBUTION.md

### PR DESCRIPTION
**Pull request by battletoilet**
#### Description of changes
Added a new task for automatically creating a component.

#### Focus areas to test
run `npm run createcomponent -- --name ExampleComponentName

### CODE REVIEW

**Comment by lynamemi** [comment]
this is so cool!

**Comment by micahgodbolt** [comment]
Very cool. Can we get a demo in the Fabric Sync today?

File: .vscode/launch.json [hunk 1]
```diff
@@ -2,6 +2,27 @@
   "version": "0.2.0",
   "configurations": [
     {
+      "name": "Debug add new component",
```
Comment (dzearing) [inline .vscode/launch.json]: can you add this to the end of the file or not mess with it? I want it to default to debug test.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Creating New Components
+
+### Creating a component via the command line
+1. Open a command prompt in the root directory of your project directory.
+2. Run `npm run createcomponent -- --name YourAwesomeNewComponentName`
+  * Our components use the Pascal Case naming convention.
+
+### Creating a component manually.
+1. From the root of the project, travel to `packages/office-ui-fabric-react/src/components/`
```
Comment (dzearing) [inline CONTRIBUTING.md]: need to refine this documentation to explain the process of adding new components.

For new ideas:
create components in `experiments`
consume in products, refine

To prep to move into oufr:
get design approval
add documentation
add tests
add vr tests
migrate to oufr

**Pull request merged**